packages/frontend-v2/components/Plan/Plan.tsx

```diff
@@ -69,6 +70,10 @@ export const Plan: React.FC<PlanProps> = ({
     mutateStudentWithUpdatedPlan(updatedPlan);
   };
 
+  useEffect(() => {
+    console.log(getPreReqWarnings(plan.schedule));
```

Comment:
just checking, did yall test getCoReqWarnings too?

Reply:
Yupp, I tested both and they are both working